setup.py

```diff
@@ -40,7 +40,9 @@ def get_version(rel_path: str) -> str:
         "fusepy>=3",
         "appdirs>=1.4.4",
         "pytimedinput>=2.0.1",
-        "click>=8.0.4"],
+        "click>=8.0.4",
```

Comment:
What's `click`?

Reply:
[Package we use for the CLI](https://github.com/pallets/click)